dces-drc-integration/build.gradle

```diff
@@ -1,36 +1,36 @@
 plugins {
 	id 'java'
 	id 'jacoco'
-	id 'org.springframework.boot' version '3.2.4'
-	id 'io.spring.dependency-management' version '1.1.4'
-	id "org.sonarqube" version "4.0.0.2929"
+	id 'org.springframework.boot' version '3.3.2'
+	id 'io.spring.dependency-management' version '1.1.6'
+	id "org.sonarqube" version "5.1.0.4882"
 }
 
 def versions = [
-		pitestVersion          : '1.4.10',
-		sentryVersion          : '6.15.0',
-		springdocVersion       : '2.3.0',
-		okhttpVersion          : '4.9.3',
+		pitestVersion          : '1.16.1',
+		sentryVersion          : '7.13.0',
+		springdocVersion       : '2.1.0',
+		okhttpVersion          : '4.12.0',
 		mockwebserverVersion   : '4.12.0',
 		reactorTestVersion     : '3.2.3',
-		springSecurityConfig   : '6.2.0',
-		springSecurityWeb      : '6.1.7',
-		springOAuthJose        : '6.1.1',
-		springOAuthClient      : '3.1.1',
-		springOAuthResource    : '3.1.1',
+		springSecurityConfig   : '6.3.1',
+		springSecurityWeb      : '6.3.1',
```

Comment:
Is there a reason these Spring project versions are given explicitly? Don't we use the `spring-boot-dependencies` [BOM](https://docs.spring.io/spring-boot/gradle-plugin/managing-dependencies.html#:~:text=Managing%20Dependencies%20with%20Gradle's%20Bom%20Support,-Gradle%20allows%20a&text=the%20dependency%20graph.-,The%20SpringBootPlugin%20class%20provides%20a%20BOM_COORDINATES%20constant%20that%20can%20be,Kotlin). Many of the versions could be updated implicitly in future if adopted.